keps/sig-node/4569-cgroup-v1-deprecation/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Deprecate Support for cgroup v1 in Kubernetes
+kep-number: 4569
+authors:
+  - "@harche"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable
+creation-date: 2024-04-04
+reviewers:
```

Comment:
Happy to help reviewer as well if you need an additional reviewer :)

Reply:
Thanks @yujuhong